locales/en/messages.json

```diff
@@ -1064,7 +1064,7 @@
     "featureGPSTip": {
         "message": "Configure port scenario first"
     },
-    "featureSONAR": {
+    "featureRANGEFINDER": {
         "message": "Sonar"
     },
```

Comment:
If for the user this is more a "Sonar", maybe is better to rename it in the firmware and not here?

Reply:
Good call, will change RANGEFINDER to SONAR in firmware but RANGEFINDER is just one possible SONAR device?